### PR TITLE
minor cleanup in runtests.py

### PR DESCRIPTION
**Pull request by twmr**
don't use mutable types in function signatures.

### CODE REVIEW

File: sympy/utilities/runtests.py [hunk 1]
```diff
@@ -242,6 +250,10 @@ def run_all_tests(test_args=(), test_kwargs={}, doctest_args=(),
     """
     tests_successful = True
 
+    test_kwargs = test_kwargs or {}
```
Comment (smichr) [inline sympy/utilities/runtests.py]: use similar above at line 205?

**Comment by smichr** [comment]
just one nit, otherwise this looks fine

File: sympy/utilities/runtests.py (hunk 1 above)
Comment (thisch) [inline sympy/utilities/runtests.py]: Do you prefer the more explicit check at line 205?

Comment (smichr) [inline sympy/utilities/runtests.py]: No. There isn't any error checking so you might as well just assign a dictionary if a null value (e.g. {} or None is sent). But maybe I am missing something that you are seeing. Is there a reason that you were explicit there but not here? If not, I would prefer to see the same `test_kwargs = test_kwargs or {}` replacing lines 205-206.

Comment (smichr) [inline sympy/utilities/runtests.py]: No. There isn't any error checking so you might as well just assign a dictionary if a null value (e.g. {} or None is sent). But maybe I am missing something that you are seeing. Is there a reason that you were explicit there but not here? If not, I would prefer to see the same `test_kwargs = test_kwargs or {}` replacing lines 205-206.

Comment (thisch) [inline sympy/utilities/runtests.py]: No, there is no reason, that I was explicit there but not here. I'll consolidat this. The only reason why I moved this into the function body is that it should be avoided to have mutable defaults in the function signature.

**Comment by smichr** [comment]
Looks good. Thanks for your attention to this. It's in.

**Pull request merged**